### PR TITLE
chore: display all pools and my pools on earn tab bar

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Changes the pools tab titles and display from:
- Open Pools & My Pools
- All Pools & My Pools

| iOS Before | iOS After |
| ----- | ----- | 
|  |  | 

### Test plan

- [x] Tested locally on iOS
- [x] Unit tests updated

### Related issues

- Fixes ACT-1332

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/earn/EarnHome.tsx [hunk 1]
```diff
@@ -238,7 +238,7 @@ export default function EarnHome({ navigation, route }: Props) {
     return pools.filter((pool) => {
       const depositTokenInfo = allTokens[pool.dataProps.depositTokenId]
       const isMyPool = new BigNumber(pool.balance).gt(0) && !!depositTokenInfo
-      return activeTab === EarnTabType.MyPools ? isMyPool : !isMyPool
+      return activeTab === EarnTabType.MyPools ? isMyPool : pools
```
Comment (finnian0826) [inline src/earn/EarnHome.tsx:241]:
```suggestion
      return activeTab === EarnTabType.MyPools ? isMyPool : true
```
this is just returning a bool for filtering so should be true always for AllPools case

Comment (finnian0826) [inline src/earn/EarnHome.tsx:241]: I think `EarnTabType` enum also should be update to be AllPools rather than OpenPools just for consistency

Comment (satish-ravi) [reply src/earn/EarnHome.tsx:241]: maybe just early bail if tab is all pools? E.g.,
```ts
return activeTab === <all pools> ? pools : pools.filter(...)
```

File: locales/base/translation.json [hunk 2]
```diff
@@ -2614,7 +2614,8 @@
     },
     "poolFilters": {
       "openPools": "Open Pools",
```
Comment (satish-ravi) [inline locales/base/translation.json:2616]: is this used any longer? if not, can we remove?

Comment (MuckT) [reply locales/base/translation.json]: Removed in ba933e128cc2dc5655a678f3534d6f669e9437b6!

File: src/earn/EarnHome.tsx (hunk 1 above)
Comment (MuckT) [reply src/earn/EarnHome.tsx]: Updated the enum in 499eb47ad2f48081a3680818340b064f644073e3!

File: src/earn/EarnHome.test.tsx [hunk 3]
```diff
@@ -159,16 +160,21 @@ describe('EarnHome', () => {
...
     )
 
+    // All Pools
     expect(
       queryByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
-    ).toBeFalsy()
+    ).toBeTruthy()
+    expect(
+      getByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
+    ).toBeTruthy()
```
Comment (satish-ravi) [inline src/earn/EarnHome.test.tsx:175]: seems like these are asserting the same thing, should these be different?

Comment (finnian0826) [reply src/earn/EarnHome.test.tsx:175]: I think should be `PoolCard/ethereum-sepolia:0xe50fa9b3c56ffb159cb0fca61f5c9d750e8128c8`?

File: src/earn/EarnHome.test.tsx [hunk 4]
```diff
@@ -159,16 +160,21 @@ describe('EarnHome', () => {
...
     fireEvent.press(getByText('earnFlow.poolFilters.myPools'))
+    // My Pools
```
Comment (finnian0826) [inline src/earn/EarnHome.test.tsx:177]: Maybe should also assert here that we don't see the Eth pool

File: src/earn/EarnHome.test.tsx (hunk 3 above)
Comment (MuckT) [reply src/earn/EarnHome.test.tsx]: Fixed in737593bcfa64e72ec22cea78c982fbbf3b03ee94!

File: src/earn/EarnHome.test.tsx (hunk 4 above)
Comment (MuckT) [reply src/earn/EarnHome.test.tsx:177]: Added in 737593bcfa64e72ec22cea78c982fbbf3b03ee94!

**Review by satish-ravi — APPROVED** [review]

**Pull request merged**